src/librustdoc/html/render/mod.rs

```diff
@@ -3682,8 +3682,9 @@ fn spotlight_decl(decl: &clean::FnDecl, cache: &Cache) -> String {
         if let Some(impls) = cache.impls.get(&did) {
             for i in impls {
                 let impl_ = i.inner_impl();
-                if impl_.trait_.def_id_full(cache).map_or(false, |d| cache.traits[&d].is_spotlight)
-                {
+                if impl_.trait_.def_id().map_or(false, |d| {
```

Comment:
Was `def_id_full` changed to `def_id` because there's no such thing as a primitive trait?

Reply:
Absolutely.